frontend/justfile

```
@@ -40,6 +40,13 @@ wait-up: up
 # Set up user and test site in Plausible
 init: wait-up
     cd .. && ./setup_plausible.sh
+# Recreate the frontend 
+recreate:
+    rm -rf node_modules
```

This will remove the node_modules inside the current directory but we actually need to remove all `node_modules/` (repo root and all subdirectories).

```suggestion
    find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +
```